packages/odyssey-react/src/components/Modal/Modal.test.tsx

```diff
@@ -130,6 +130,18 @@ describe("Modal", () => {
     expect(handleClose).toHaveBeenCalledTimes(1);
   });
 
+  xit("should automatically focus on the dismiss icon when modal is open", () => {
+    const handleClose = jest.fn();
+    const { container } = render(
+      <Modal open onClose={handleClose} closeMessage={message}>
```

Comment:
I'm not sure why the `onClose` prop is required for the Modal component -- it seems we should make that optional.

Reply:
hmmm I'm unsure myself - my guess is because the modal is driven via props the onClose callback is required to help enforcing folks updating their own modal open/closed state in the outer component.

It may be worth taking 5 minutes to determine if the component will always need to be driven through externalized state like this or if we can instead offer an uncontrolled component API experience as an alternative

Reply:
+1 on the `onClose` being required because the Modal is controlled externally and wanting to be explicit about having to handle the close interactions.